backend/api/serializers/project_serializer.py

```diff
@@ -33,6 +34,22 @@ def to_representation(self, instance: Project):
         if (groups_submitted > non_empty_groups):
             non_empty_groups = groups_submitted
 
+        extra_checks_count = instance.extra_checks.count()
+
+        if extra_checks_count:
+            passed_extra_checks_submission_ids = ExtraCheckResult.objects.filter(
+                submission__group__project=instance,
+                submission__is_valid=True,
+                result=StateEnum.SUCCESS
+            ).values_list('submission__id', flat=True)
+
+            passed_extra_checks_group_ids = Submission.objects.filter(
+                id__in=passed_extra_checks_submission_ids
+            ).values_list('group_id', flat=True)
+
+            unique_groups = set(passed_extra_checks_group_ids)
+            extra_checks_passed = len(unique_groups)
```

Comment:
If you're bored, I'm fairly certain this can be optimized as in terms of the amount of db calls.
However I'm good to leave it like that

Reply:
:earth_asia: